client/data/settings/selectors.js

```diff
@@ -201,7 +201,6 @@ export const getPlatformCheckoutLocations = ( state ) => {
 	);
 };
 
-// TODO: Finish implementation once server API endpoints are deployed.
 export const getCurrentProtectionLevel = ( state ) => {
 	return getSettings( state ).current_protection_level || 'standard';
```

Comment:
What do you think about moving the `standard` to a constant?

Reply:
Moved in [773170d](https://github.com/Automattic/woocommerce-payments/pull/5685/commits/773170da25e7fd0251de70ebc538c9f68c15d8bf).